Nodejs/Product/Npm/SPI/NpmGetCatalogCommand.cs

```diff
@@ -120,7 +120,10 @@ internal class NpmGetCatalogCommand : NpmCommand, IPackageCatalog {
                                             .FirstOrDefault();
 
                                         if (!string.IsNullOrEmpty(latestVersion)) {
-                                            builder.LatestVersion = SemverVersion.Parse(latestVersion);
+                                            try {
+                                                builder.LatestVersion = SemverVersion.Parse(latestVersion);
+                                            } catch (SemverVersionFormatException) { 
```

Comment:
Also print out a message here - we want to know about the issue, but don't want it to block the user.